core/src/test/scala/org/apache/spark/scheduler/TaskSchedulerImplSuite.scala

```diff
@@ -140,6 +140,16 @@ class TaskSchedulerImplSuite extends SparkFunSuite with LocalSparkContext with B
     taskScheduler
   }
 
+  private def submitTasksAndCheck(
+      offers: IndexedSeq[WorkerOffer],
```

Comment:
I found that many tests use the same WorkerOffer, e.g., `WorkerOffer("executor0", "host0", 1)`. Shall we make it as the default value as well?

Reply:
OK